RELEASE_CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 ### [v2.1.0](https://github.com/provenance-io/provwasm/tree/v2.1.0)
 
 * Update to provenance 1.16 ([#118](https://github.com/provenance-io/provwasm/issues/118))
```

Comment:
Is this still correct? I know we have a release candidate for 1.17.

Reply:
good catch, fixed now